db/seeds.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+secret_key = ENV['STRIPE_SEEDS_SECRET_KEY']
+publishable_key = ENV['STRIPE_SEEDS_PUBLISHABLE_KEY']
+
+puts "Loading seed: solidus_stripe/stripe_payment_method"
+
+if secret_key.blank? || publishable_key.blank?
+  puts "Failure: You have to set both STRIPE_SEEDS_SECRET_KEY and STRIPE_SEEDS_PUBLISHABLE_KEY environment variables."
+else
+  stripe_payment_method = Spree::PaymentMethod::StripeCreditCard.new do |gateway|
+    gateway.name = 'Credit Card'
+    gateway.preferred_server = "test"
+    gateway.preferred_test_mode = true
```

Comment:
Should we make this 

```rb
gateway.preferred_test_mode = !Rails.env.production?
```

Reply:
I think in production is better to use `static_model_preferences` so making this available in production as well could encourage a bad practice. 

Unfortunately, I can't figure out a way to use `static_model_preferences` with seeds, since it would require something into the initializer and I'd like to keep all the seed configuration in one place. We could create an initializer within the stripe installation but I'd like to keep this as simple as possible. What do you think?